src/components/CommentCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function CommentCard ({comment}){
+
+    let votes = comment.votes
+    
+    let voteType = votes < 0 ? "dislikes" : "likes";
+    return (
+    
+        <div className="CommentCard">
+            <h3>{comment.author}: {comment.body}</h3>
+            <h4>{votes} {voteType}</h4>
+            <h5>Created: {comment.created_at}</h5>
```

Comment:
You could make this date a bit more readable, either yourself using string manipulation or a library like https://day.js.org/